index.html

```diff
@@ -1453,45 +1444,13 @@
 
 </pre>
         <p>
-          The <dfn for="Presentation"><code>defaultRequest</code></dfn> MUST
-          return the <a>default presentation request</a> if any, null
-          otherwise.
+          The <a>PresentationReceiver</a> object is available to a <a>presenting
```

Comment:
The spec should convert to use the terminology "receiving browsing context" to align API naming and API concepts.